### PR TITLE
[NCL-7418] Update typescript, lerna and rollup

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: packages/umd-bundle/dist/pnc-js-libs.js [hunk 1]
```diff
@@ -176,6 +178,4 @@
 
     exports.MessageBus = MessageBus;
 
-    Object.defineProperty(exports, '__esModule', { value: true });
```
Comment (matedo1) [inline packages/umd-bundle/dist/pnc-js-libs.js:179]: rollup v1.32.1:
https://github.com/rollup/rollup/blob/v1.32.1/docs/999-big-list-of-options.md#outputesmodule

rollup v3.3.0:
https://github.com/rollup/rollup/blob/v3.3.0/docs/999-big-list-of-options.md#outputesmodule

**Comment by matedo1** [comment]
Hi @patrikk0123 , can you confirm that 

- build steps: https://github.com/project-ncl/pnc-js-libs#build-project
- release steps: https://github.com/project-ncl/pnc-js-libs#release

are working ok?

**Comment by patrikk0123** [comment]
> Hi @patrikk0123 , can you confirm that
> 
> * build steps: https://github.com/project-ncl/pnc-js-libs#build-project
> * release steps: https://github.com/project-ncl/pnc-js-libs#release
> 
> are working ok?

Tested and both of these work.

**Review by DnsZhou — APPROVED** [review]
I tested both the build and the release process, all passed without issue. Thanks Patrik!

**Pull request merged**